jobs/email/EditorialReminder.php

```diff
@@ -81,7 +81,7 @@ public function handle(): void
 
         /** @var int $submissionId */
         foreach ($submissionIds as $submissionId) {
-            $submission = Repo::submission()->get($submissionId);
+            $submission = Repo::submission()->get((int)$submissionId);
             $submissions[$submissionId] = $submission;
 
             if ($submission->getData('stageId') === WORKFLOW_STAGE_ID_SUBMISSION) {
```

Comment:
This could be one of those weird data type issues, I think. `$submission->getData('stageId')` might come back from the DB as a string. To be safe, use a `==` comparison.

Reply:
Done .